src/evaluationUtils/buildContext.ts

```diff
@@ -48,7 +54,7 @@ export default function buildEvaluationContext(
 	expressionString: string,
 	contextItem: any,
```

Comment:
Let's consider making the 'ContextItem' also a `TypedExternalValue|UntypedExternalValue` Removes another `any`